kore/test/Test/Kore/Step/Simplification/IntegrationProperty.hs

```diff
@@ -72,7 +72,7 @@ test_simplifiesToSimplified =
         simplified <- catch
             (evaluateT (Pattern.fromTermLike term))
             (exceptionHandler term)
-        (===) True (OrPattern.isSimplified sideRepresentation simplified)
+        (===) True (OrPattern.hasSimplifiedChildren sideRepresentation simplified)
```

Comment:
@ttuegel Based on the discussion during the meeting, this should actually be `isSimplified`, right? Further down in this file, we're using the `Pattern` simplifier to simplify this, so we should make sure that the whole `Pattern` is simplified at the end.